packages/common/src/service.js

```diff
@@ -52,9 +57,20 @@ function Service(config, options) {
     email: options.email
   });
 
+  if (!config.baseUrl) {
```

Comment:
@stephenplusplus - This is temporary until I wrap up with the other services. Without this the integration tests break. Sorry it's taking this long...